src/main/java/com/modern/training/platform/config/LoggingAspectConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.modern.training.platform.config;
+
+import java.util.Arrays;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class LoggingAspectConfig {
+
+	Logger log = LoggerFactory.getLogger(this.getClass());
```

Comment:
While this is fine for a start, I'd suggest we later replace these with Lombok's @slf4j annotation. 
For further reading: [Lombok's slf4j](https://projectlombok.org/api/lombok/extern/slf4j/Slf4j.html)